.github/release-template.md

```diff
@@ -0,0 +1,18 @@
+This week's push hero is @{{GITHUB_USER}}
+
+Latest Release: [{{LATEST_TAG}}]({{LAST_RELEASE_URL}})
```

Comment:
Isn't this always going to be wrong, if we're creating the release notes before the tag? Or is the idea we'd update it manually afterwards?

Reply:
Latest release points to the most recent release tagged "latest" which all of them are, so effectively it is the "last" release. Each time you create a new release, it links to the previous release. Should we call it "previous" instead of "latest"?

Reply:
Previous is going to be correct every time we're creating a draft release, yeah.

Reply:
It's not clear if you are asking for a change or not? Is it fine to leave latest, or should I update to say "previous"?

Reply:
I think it should be changed to previous release, at least for draft releases.